website/content/docs/installing/postgres.mdx

```diff
@@ -16,4 +16,20 @@ Boundary has only been tested on Postgres 11 or greater.
 
 ## Network
 
-Boundary controllers must be able to reach Postgres. In non-HA configurations, this means Boundary servers; if you're running in [high availability](/docs/installing/high-availability), then the controllers need access to the Postgres server infrastructure. Worker nodes never need access to the database.
\ No newline at end of file
+Boundary controllers must be able to reach Postgres.
+In non-HA configurations, this means Boundary servers;
```

Comment:
Did you want a semicolon here or a period?

Reply:
I only reformatted that line.